src/Donations/Components/DonationsForm.tsx

```diff
@@ -216,78 +224,77 @@ function DonationsForm() {
       default:
         paymentLabel = t("treesInCountry", {
           treeCount: quantity,
-          country: t(`country:${projectDetails.country?.toLowerCase()}`),
+          // country: t(`country:${projectDetails.country?.toLowerCase()}`),
```

Comment:
if these line of code not needed then remove it from here

Reply:
Done in - 2b80002f15975bc7214694c3593e86ec6047643f